e2e/specs/cookie.spec.js

```diff
@@ -0,0 +1,26 @@
+const SingIn = require('../pages/SignIn');
+const CookieManager = require('../cookies/CookieManager');
+let username = 'hapsneeze';
+let password = 'test12345';
+describe('pivotal tracker verify user cookie', () => {
+    let dashboard;
+
+    before(() => {
+        dashboard = SingIn.loginAs(username, password);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "dashboard" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)